### ISSUE TITLE
[Feature request] Show copied text on notification

### ISSUE THREAD

**Issue by alespg** [Tudmotu/gnome-shell-extension-clipboard-indicator#193]
Hi,
I would like to request a feature for this extension, would it be possible that the notification show the text (in case it is text) which is copied to clipboard?
i know it may seems useless, but i would like to have that option for some reason i dont know very well.

Thank you for making this extension, it has make my life easier and happier :)

**Comment by Tudmotu** [comment]
I don't believe he have such an option right now, but you can display the currently-copied text in the indicator - there's an option in the settings. Does that help? :)

**Comment by alespg** [comment]
thanks @Tudmotu 
yeah, i know, but no, not really. i dont show the text there because of the space it takes, but i think it would not be very difficult, if you have already the information when when you show the notification, i will give it a try when i have some time

**Comment by alespg** [comment]
I have made the changes, but i dont have permission to push my changes:

line ~482

                if (NOTIFY_ON_COPY) {
                    that._showNotification(_(text), notif => {
                        notif.addAction(_('Cancel'), Lang.bind(that, that._cancelNotification));
                    });

line ~592
        if (this._notifSource.count === 0) {
            notification = new MessageTray.Notification(this._notifSource, 'Copied to clipboard', message);
        }

It works fine (at least as i expected)

**Comment by alespg** [comment]
I have made other changes in notifications:

When you copy some text:
![Screenshot from 2020-02-10 07-15-57](https://user-images.githubusercontent.com/53279202/74125498-c7c64b00-4bd5-11ea-8424-01920f0a752d.png)

When you cycle clipboard history:
![Screenshot from 2020-02-10 07-17-59](https://user-images.githubusercontent.com/53279202/74125497-c72db480-4bd5-11ea-92c2-506f876e29be.png)

[extension.js.txt](https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator/files/4179236/extension.js.txt)

**Comment by Tudmotu** [comment]
Nice! :)
To send a Pull Request, you need to "fork" this repository, and push your changes to your fork. then, you will have a button "open a pull request". Once you open a pull request, I can review this change.

**Comment by alespg** [comment]
yes, google is my friend :)

thank you for letting me participate in this project, it is one of my favourites extensions and it is quite exciting

anyway, be careful with the pull request, it is likely to have some error

**Comment by user-1-2-3-4-5** [comment]
Android has displayed a notification (or a floating window to be exact) with the copied text for years. This feature would be of use, since the terminal and graphical programs have their own key combinations and I tend to subconsciously hit the wrong one (or mispress a key).
